apps/vue-showcase/cypress/e2e/vue-route-guard.cy.ts

```diff
@@ -0,0 +1,66 @@
+describe(`@this-dot/vue-route-guard`, () => {
+  describe(`No authentication set`, () => {
+    before(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.removeItem('VUEROUTEGUARD-TOKEN');
+      });
+      cy.clearLocalStorage('VUEROUTEGUARD-TOKEN');
+      cy.clearCookie('VUEROUTEGUARD-TOKEN');
+    });
+
+    it(`redirect to login`, () => {
```

Comment:
I'm just nitpicking here, but the test is in third person singular, so it should be ```it(`redirects to login`, () =>{})``` to make it grammatically correct. :)